addons/dragging/stringtable.xml

```diff
@@ -86,21 +86,9 @@
             <Chinesesimp>背起</Chinesesimp>
             <Turkish>Taşı</Turkish>
         </Key>
-        <Key ID="STR_ACE_Dragging_LowerRaise">
-            <English>Raise/Lower</English>
-            <German>Heben/Senken</German>
-            <Polish>Wyżej/Niżej</Polish>
-            <Portuguese>Levantar/Abaixar</Portuguese>
-            <Russian>Поднять/Опустить</Russian>
-            <Czech>Zvýšit/Snížit</Czech>
-            <Italian>Alza/Abbassa</Italian>
-            <Spanish>Subir/Bajar</Spanish>
-            <French>Lever/Baisser</French>
-            <Japanese>上げる/下げる</Japanese>
-            <Korean>높이기/낮추기</Korean>
-            <Chinese>提高/下降</Chinese>
-            <Chinesesimp>提高/下降</Chinesesimp>
-            <Turkish>Yukarı/Aşağı</Turkish>
+        <Key ID="STR_ACE_Dragging_RaiseLowerRotate">
+            <English>Raise/Lower | (Ctrl + Scroll) Rotate</English>
+            <German>Heben/Senken | (Strg + Scrollen) Drehen</German>
```

Comment:
```suggestion
            <German>Heben/Senken | (Strg + Scrollrad) Drehen</German>
```

Scroll is not a key. It shoul

Reply:
Same as here:
https://github.com/acemod/ACE3/blob/master/addons/advanced_throwing/stringtable.xml#L247

Reply:
Then change it there as well.

Reply:
No. Can be done in a separate PR.

Reply:
No, do not change it. Scroll is scroll wheel rolling there is also a Middle Mouse Button as key which is also used in Advanced Throwing.